### ISSUE TITLE
Cannot install kf5-kdoctools

### ISSUE THREAD

**Issue by RichardsonAlex** [haraldF/homebrew-kf5#135]
This is possibly related to the sandbox?

```
==> Installing kf5-kdoctools from haraldf/kf5
==> Downloading http://download.kde.org/stable/frameworks/5.37/kdoctools-5.37.0.
Already downloaded: /Users/alex/Library/Caches/Homebrew/kf5-kdoctools-5.37.0.tar.xz
==> Patching
==> cpanm URI
Last 15 lines from /Users/alex/Library/Logs/Homebrew/kf5-kdoctools/01.cpanm:

cpanm
URI

--> Working on URI
Fetching http://www.cpan.org/authors/id/E/ET/ETHER/URI-1.72.tar.gz ... OK
Configuring URI-1.72 ... OK
==> Found dependencies: Test::Needs
--> Working on Test::Needs
Fetching http://www.cpan.org/authors/id/H/HA/HAARG/Test-Needs-0.002005.tar.gz ... OK
Configuring Test-Needs-0.002005 ... OK
Building and testing Test-Needs-0.002005 ... ! Installing Test::Needs failed. See /private/tmp/kf5-kdoctools-20170829-601-yj56xv/kdoctools-5.37.0/.brew_home/.cpanm/work/1504005473.617/build.log for details. Retry with --force to force install it.
! Installing the dependencies failed: Module 'Test::Needs' is not installed
! Bailing out the installation for URI-1.72.
FAIL

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/haraldf/homebrew-kf5/issues

These open issues may also help:
kf5-kdoctools.rb looking for inexistant "/usr/local/opt/python3/bin/python3.5" https://github.com/haraldF/homebrew-kf5/issues/81
```

**Comment by RichardsonAlex** [comment]
This workaround "fixed" it for me:
```diff
diff --git a/kf5-kdoctools.rb b/kf5-kdoctools.rb
index 6d41ddb..e0496aa 100644
--- a/kf5-kdoctools.rb
+++ b/kf5-kdoctools.rb
@@ -22,7 +22,7 @@ class Kf5Kdoctools < Formula
   end

   def install
-    system "cpanm", "URI"
+    # system "cpanm", "URI"

     args = std_cmake_args
```

**Comment by RichardsonAlex** [comment]
Didn't see this is a duplicate of #126 

So I guess the solution is to add a message that the user has to run `cpanm URI` manually

**Comment by yurikoles** [comment]
@RichardsonAlex fixed in 83d6d06c72b81b53f645b2465c69f57e74ab95cd
